implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OtlpDeltaMeterRegistryTest.java

```diff
@@ -24,42 +24,55 @@
 
 import java.time.Duration;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Function;
 
 import static io.micrometer.registry.otlp.AggregationTemporality.DELTA;
 import static org.assertj.core.api.Assertions.assertThat;
 
-class DeltaOtlpMeterRegistryTest {
+class OtlpDeltaMeterRegistryTest extends OtlpMeterRegistryTest {
```

Comment:
There are lots of changes in this PR that are not necessarily needed for its goal:
`Improve Otlp Delta Aggregation with support for max and Histogram`, can we separate these?

Reply:
You mean the tests? Other than tests(even there only I did some refactoring to remove some of the duplication) all the changes are kind lf essential. Let me know if any specifics you think should be addressed separately

Reply:
Definitely not the tests, I'm sorry I Made this comment on the wrong class, I wanted to comment on one of the consolidated meters. :)
I meat using OTel's AggregationTemporality and consolidating the separated cumulative/step meters but it seems both are fixed now in: https://github.com/micrometer-metrics/micrometer/pull/3749/commits/b446f0514956e691bccefb6274c7c525e1415bfd